middleware/fileserver.go

```diff
@@ -128,6 +129,10 @@ func (fh *fileHandler) serveFile(w http.ResponseWriter, r *http.Request, name st
 		return http.StatusNotFound, nil
 	}
 
+	// Add ETag header
+	e := fmt.Sprintf("\"%x-%x\"", d.ModTime().Unix(), d.Size())
```

Comment:
It's a small thing, but I really don't like exposing more information than necessary.  While the last modification time may also be returned by the http request, putting it in here seems "wrong".

However, that can be changed later on.  :)